5calls/app/src/main/java/org/a5calls/android/a5calls/controller/IssueActivity.java

```diff
@@ -202,6 +206,7 @@ protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
         outState.putParcelable(KEY_ISSUE, mIssue);
         outState.putBoolean(KEY_IS_LOW_ACCURACY, mIsLowAccuracy);
+        outState.putBoolean(KEY_IS_STARRED, mIsStarred);
```

Comment:
If we annotate `mIssue` with `isStarred` you wouldn't need to set/get this in the state.